### PR TITLE
Form create

### PR DESCRIPTION
**Pull request by AlexPro100pro**
[DEMO LINK](https://AlexPro100pro.github.io/layout_html-form/)
[TEST REPORT LINK](https://AlexPro100pro.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,194 @@
...
+        </div>
+        <div class="form-field">
+          <label for="cars">
+            What are your favorite brand of cars?
+          </label>
+          <select
+           name="cars"
+           multiple
+           id="cars"
+          >
```
Comment (ArturErmolenko) [inline src/index.html:158]:
```suggestion
          <select
            name="cars"
            multiple
            id="cars"
          >
```

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,194 @@
...
+          <label for="comment">
+            Comments:
+          </label>
+          <textarea
+            name="comment"
+            cols="20"
+            rows="2"
+            id="comment"
+            maxlength="250"
+          >
```
Comment (ArturErmolenko) [inline src/index.html:187]: checklist point 8

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Good job 👍

File: src/index.html (hunk 2 above)
Comment (AlexPro100pro) [reply src/index.html]: ready

**Pull request closed without merging**